packages/c-table/src/c-table-container.tsx

```diff
@@ -0,0 +1,36 @@
+import { h, defineComponent, PropType } from "vue"
+import {
+  chakra,
+  ComponentWithProps,
+  DeepPartial,
+  HTMLChakraProps,
+} from "@chakra-ui/vue-system"
+
+export interface CTableContainerProps extends HTMLChakraProps<"div"> {}
+
+export const CTableContainer: ComponentWithProps<
+  DeepPartial<CTableContainerProps>
+> = defineComponent({
+  name: "CTableContainer",
+  props: {
+    overflow: String as PropType<CTableContainerProps["overflow"]>,
+    overflowX: String as PropType<CTableContainerProps["overflowX"]>,
+  },
+  setup(props, { slots, attrs }) {
+    return () => (
+      <chakra.div
+        __css={{
+          display: "block",
+          whiteSpace: "nowrap",
+          WebkitOverflowScrolling: "touch",
+          overflowX: props.overflow ?? props.overflowX ?? "auto",
+          overflowY: "hidden",
+          maxWidth: "100%",
+        }}
+        {...attrs}
+      >
+        {slots}
```

Comment:
Prefer rendering {slots?.default?.()}

https://vuejs.org/guide/extras/render-function.html#rendering-slots:~:text=%5D%0A%20%20%7D%0A%7D-,JSX%20equivalent%3A,-jsx

Reply:
That's an old whoops of mine! 😅